viewing_party/party.py

```diff
@@ -1,23 +1,138 @@
+from re import T
+
 # ------------- WAVE 1 --------------------
 
+# This function adds 3 key-value pairs to a dictionary called movie_dict
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    if title and genre and rating:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+    else:
+        return None
+
+# This function adds a key-value pair called "watched" to a dictionary called user_data; It includes all movies that the user has watched 
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]

```

Comment:
This passes the tests but is a bug.  

Since you used `=` instead of appending `movie` this will remove all of the previously watched movies:
```suggestion
    user_data["watched"].append(movie)

```